lib/myfinance/resources/payable_account.rb

```diff
@@ -8,27 +8,21 @@ module Resources
     #
     class PayableAccount < FinancialAccount
 
-      def pay(id, entity_id, params = {})
-        http.put(endpoints(id, entity_id, :pay), body: { payable_account: params }) do |response|
-          respond_with_object(response, 'payable_account')
-        end
-      end
+      private
 
-      def undo_payment(id, entity_id)
-        http.put(endpoints(id, entity_id, :undo_payment)) do |response|
-          respond_with_object(response, 'payable_account')
-        end
+      def endpoints
+        default_endpoints.merge({
+          pay: '/entities/:entity_id/payable_accounts/:id/pay',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.